webclient/src/types/server.tsx

```diff
@@ -54,6 +54,12 @@ export class Host {
 }
 
 export const DefaultHosts: Host[] = [
+  {
```

Comment:
So, politics time.. the dropdown will be ordered by this list. Rooster seems to be the main server, which is why I have it listed first, as it will be selected automatically by default.

Otherwise, this looks fine. You might want to play with the `localHost` value, and try host and localHost with and without /servatrice to see if it works locally with `ws`.

Reply:
every other server list has been alphabetical.
Yeah ill give that localHost a try